### PR TITLE
Added test case to validate TimestampRangeFilter with Integer.MAX_VALUE

### PR DESCRIPTION
**Pull request by rupeshitpatekarq**
Test case to validate TimestampRangeFilter with Integer.MAX_VALUE

### CODE REVIEW

**Comment by sduskis** [comment]
This test requires adding data to a table, and then actually performing a `Get` on the table and returning data.  Seeing that a filter exists is not enough.

**Comment by rupeshitpatekarq** [comment]
Updated test case as per review comment. Please confirm.

File: bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java [hunk 1]
```diff
@@ -95,6 +96,43 @@ public void testBigtableFilter() throws IOException {
...
+  
+  /**
+   * This test case is used to validate TimestampRangeFilter with Integer.MAX_VALUE #1552
+   * 
+   * @throws IOException
+   */
+  @Test
+  public void testTimestampRangeFilterWithMaxVal() throws IOException {
+	    // Initialize
+	    int numCols = Integer.MAX_VALUE;
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java]: Were you actually able to run this?  numCols with MAX_VALUE should crash any tests.

File: bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java [hunk 2]
```diff
@@ -95,6 +96,43 @@ public void testBigtableFilter() throws IOException {
...
+	    Long start = (long) (Integer.MAX_VALUE - 2);
+	    String goodValue = "includeThisValue";
+	    Table table = getDefaultTable();
+	    byte[] rowKey = dataHelper.randomData("testRow-TimestampRange-");
+	    Put put = new Put(rowKey);
+	    for (Long i = start; i < numCols; ++i) {
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java]: `Long i` should be `long i`. Please inline start and numCols here.

File: bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java [hunk 3]
```diff
@@ -95,6 +96,43 @@ public void testBigtableFilter() throws IOException {
...
+	      put.addColumn(COLUMN_FAMILY, dataHelper.randomData(""), i, Bytes.toBytes(goodValue));
+	    }
+	    table.put(put);
+
+	    // Filter for results
+	    FilterList filter = new FilterList(FilterList.Operator.MUST_PASS_ALL,
+				  new TimestampRangeFilter(start, Integer.MAX_VALUE),
+				  new FamilyFilter(CompareOp.EQUAL, new BinaryComparator(Bytes.toBytes("test_family"))));
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java]: Please only use the `TimestampRangeFilter` and not the `FilterList` for clarity.

**Comment by rupeshitpatekarq** [comment]
Implemented changes & yes I ran all tests successfully in both HBase and Bigtable mode.

**Comment by sduskis** [comment]
Can you please copy the test into the 2.x version of `TestFilters` as well?  After that, this PR should be good to go.

**Comment by rupeshitpatekarq** [comment]
Added test into 2.X version of TestFilters & also ran all the tests successfully in both HBase and Bigtable mode.

**Pull request merged**